app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+    <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<hr class="margin-y-4" />
+
+<div class="grid-row">
+    <div class="grid-col-12 tablet:grid-col-3">
+        <%= image_tag(
+              asset_url('idv/phone-icon.svg'),
+              alt: t('image_description.camera_mobile_phone'),
+              width: 100,
+              height: 100,
+            ) %>
+    </div>
+    <div class="grid-col-12 tablet:grid-col-9">
+        <div class="usa-tag bg-primary-dark margin-top-1">
```

Comment:
The dark blue here isn't one of our standard tag colors. I see we used the "informative" (dark green) variant in an earlier design iteration. Was there a reason we chose a non-standard blue in later iterations?

```suggestion
        <div class="usa-tag usa-tag--informative">
```

Reply:
Note: I had a chat with @Kamal-Munshi in the Figma design, seems like something had got mixed up from earlier iterations, and it should be the "informative" green.